buildenv/ansible/roles/openj9-build-environment/tasks/main.yml

```diff
@@ -1,5 +1,5 @@
 ###############################################################################
-# Copyright (c) 2018, 2018 Pavel Samolysov
+# Copyright (c) 2018, 2021 Pavel Samolysov
```

Comment:
I think the IBM copyright date on line 22 should be updated instead of changing this.

Reply:
Updated.